site/pom.xml

```diff
@@ -39,30 +39,6 @@
     </distributionManagement>
     <build>
         <plugins>
-            <!-- copy source codes into vue required folders-->
```

Comment:
Hi, why you thought this is duplicated...
have you tried `mvn package -pl site -P site` ?

Reply:
You are right,  i mistake it